.drone.yml

```diff
@@ -1182,6 +1182,6 @@ depends_on:
 
 ---
 kind: signature
-hmac: 906ceab2f0ec44b6e9c99f698feeb4874c503fd9db2a8525ed2916445069c403
+hmac: 6f9e82fb6d1fc4166a76b998fecb456cb5e931f5c2914d023008e30fba1c8822
```

Comment:
I got it. The maintainer need to sign-off the modified YAML file each time it changes.

Reply:
Yup